website/sidebars.json

```
@@ -9,6 +9,13 @@
       "lms-connection",
       "web-analytics"
     ],
-    "Contributing": ["contributing-guide", "accessibility-testing", "css-guidelines"]
+    "Contributing": [
+      "installation",
+      "docker-development",
+      "native-installation",
```

Hmm I'll put `installation` in the `Getting started` section. Then maybe move `native-installation` into `Recipes` section.

```suggestion
      "docker-development",
```

---

The idea was to differentiate someone wanting to do a site with richie from someone wanting to contribute to richie. If You want to build your site (99% of the people), you just need cookiecutter... installation is only for developpers who want to contribute to richie...